src/main/java/seedu/duke/constants/ErrorMessage.java

```diff
@@ -53,4 +53,5 @@ public interface ErrorMessage {
     String NETWORK_OR_UNEXPECTED_ERROR = "The API could not be loaded! Please check your Internet connection " +
             "and try again. If problems persist, please report this error to the MoneyMoover team!";
     String OUTDATED_RATES = "In the meantime, we will use recently saved exchange rates!";
+    String DESCRIPTION_TOO_LONG = "Please enter a description with no more than 100 characters";
```

Comment:
I think this statement is a bit misleading. It sounds like the description text is compulsory but limited to 100 characters. Maybe we can rephrase it in a less ambiguous manner such as

```suggestion
    String DESCRIPTION_TOO_LONG = "Please limit your description to 100 characters";
```